static/app/views/eventsV2/landing.tsx

```diff
@@ -42,10 +41,8 @@ const SORT_OPTIONS: SelectValue<string>[] = [
 
 type Props = {
   organization: Organization;
-  location: Location;
-  router: ReactRouter.InjectedRouter;
-  params: Params;
-} & AsyncComponent['props'];
+} & RouteComponentProps<{}, {}> &
```

Comment:
```suggestion
} & Pick<RouteComponentProps<{}, {}>, 'location'> &
```

Reply:
It needs `router` and `params` as well here.

I realize I am kindof inconsistent about pick, but I'm more concerned about just using the same `RouteComponnetProps` everywhere.